andela/isildur-rc
# #151207883 Static Page Management

#### What does this PR do?
- Adds static page management to Reaction commerce.

#### Description of Task to be completed?
- Create and edit static pages, such as Contact, About, or FAQ/Support.

#### How should this be manually tested?
- Sign in as an admin, click the page management button to see the WYSIWYG page editor.

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/151207883

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2017-10-03 at 1 42 58 pm" src="https://user-images.githubusercontent.com/8125356/31125623-ce3b35b2-a840-11e7-8b89-98ffb9c212aa.png">
